app/frontend/src/components/apiAccess/ApiAccess.vue

```diff
@@ -45,11 +52,24 @@ export default {
     Step3,
     Step4
   },
+  props: ['acronym'],
   computed: {
     ...mapGetters('apiAccess', ['step'])
   },
   methods: {
-    ...mapMutations('apiAccess', ['setStep'])
+    ...mapMutations('apiAccess', ['setStep', 'setAcronym']),
+    ...mapActions('apiAccess', ['setStep', 'getAcronymClientStatus'])
+  },
+  created() {
+    this.$store.registerModule('apiAccess', apiAccess);
```

Comment:
I'm going to suggest we register and unregister the apiAccess state module only in the view layer. This makes it so that we know that when we are looking at a specific route view, we know the Vuex state is mounted and available for any children components.

As an extension, for every registerModule, there must also be an equivalent unregisterModule in the beforeDestroy lifecycle. Reference the `MyApps.vue` view for an example. In this case, I believe this registerModule will need to reside under the `Application.vue` view, and you'll need to add the equivalent unregisterModule on the beforeDestroy lifecycle for it.

Reply:
Ok I'll try that out

Reply:
Works nicely. Thanks!